### ISSUE TITLE
tinylog 1.0.1 not processing Java system properties correctly. 

### ISSUE THREAD

**Issue by mottati** [pmwmedia/tinylog#12]
I noticed a problem in how tinylog processes Java system properties. When (the equivilant of) -Dtinylog.level=debug is passed in on the command line, it does not appear to be processed correctly. Here is a test that reproduces the problem. 

```java
package org.tinylog.bugreport;

import org.junit.Test;
import org.pmw.tinylog.Logger;

import static org.junit.Assert.assertEquals;

public class SystemPropertyTest {

    @Test
    public void loggers() {
        String tinylog_level = System.getProperty("tinylog.level");
        Logger.info("tinylog.level=" + tinylog_level);
        Logger.debug("Log level debug");
        assertEquals(tinylog_level,Logger.getLevel().name());
    }
}
```

Here is the gradle script that I used to compile and run the test. 

```groovy
apply plugin: 'java'

String _logLevel = hasProperty('logLevel') ? project.properties.get('logLevel'): 'info'

repositories {
    jcenter()
}

dependencies {
    testCompile 'junit:junit:4.12'
    testCompile 'org.tinylog:tinylog:1.0.1'
}
test {
    testLogging {
        events "passed", "skipped", "failed", "standardOut", "standardError"
    }
    // Pass in upshifted logging level as a Java System Property to the test.
    systemProperty 'tinylog.level', _logLevel.toUpperCase()
}
```

Using the above here is the result of the test when logLevel info is passed in, followed by logLevel debug.

**tinylog.level=info**

```
gradle -PlogLevel=info clean test 
:clean
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestJava
:processTestResources UP-TO-DATE
:testClasses
:test

org.tinylog.bugreport.SystemPropertyTest > loggers STANDARD_OUT
    2015-09-29 11:30:05 [Test worker] org.tinylog.bugreport.SystemPropertyTest.loggers()
    INFO: tinylog.level=INFO

org.tinylog.bugreport.SystemPropertyTest > loggers PASSED

BUILD SUCCESSFUL

Total time: 1.523 secs
```

**tinylog.level=debug**

```
gradle -PlogLevel=debug clean test 
:clean
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestJava
:processTestResources UP-TO-DATE
:testClasses
:test

org.tinylog.bugreport.SystemPropertyTest > loggers STANDARD_OUT
    2015-09-29 11:31:07 [Test worker] org.tinylog.bugreport.SystemPropertyTest.loggers()
    INFO: tinylog.level=DEBUG

org.tinylog.bugreport.SystemPropertyTest > loggers FAILED
    org.junit.ComparisonFailure at SystemPropertyTest.java:15

1 test completed, 1 failed
:test FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///Users/michael.ottati/z-tmp/tinylog/build/reports/tests/index.html

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 1.57 secs
```

When I go back to the 0.8.1 version I had been using before (modifying the Logger.getLevel() call) this test works as expected, it passes at both tinylog.levels.

I could not step it though the initialization to see what happens there, it looks like this library ships without debug symbols. 

Here are the first copule of lines of the failure exception in the gradle/JUnit html report that gets created by the build.

```
org.junit.ComparisonFailure: expected:<[DEBUG]> but was:<[INFO]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at org.tinylog.bugreport.SystemPropertyTest.loggers(SystemPropertyTest.java:15)
```

Thanks for all the work that you do on this package, it has come in very handy where I work.

**Comment by pmwmedia** [comment]
Thanks to your detailed description I could identify the bug easily. I have fixed the code in my commit ed52182e7b7955af5217b8ea05a498fdb2c2daca. Can you test it? http://www.tinylog.org/downloads/tinylog-1.0.2.zip. I will publish the official version 1.0.2 soon.

**Comment by mottati** [comment]
Done. The test now passes. 

I made the following change to the dependency section of my build file to test your change:

```
dependencies {
    testCompile 'junit:junit:4.12'
    // testCompile 'org.tinylog:tinylog:1.0.1'
    testCompile files('lib/tinylog.jar')
}
```

The result is shown below: 

```
./gradlew clean test  -PlogLevel=debug 
:clean
:compileJava UP-TO-DATE
:processResources UP-TO-DATE
:classes UP-TO-DATE
:compileTestJava
:processTestResources UP-TO-DATE
:testClasses
:test

org.tinylog.bugreport.SystemPropertyTest > loggers STANDARD_OUT
    2015-09-29 23:20:44 [Test worker] org.tinylog.bugreport.SystemPropertyTest.loggers()
    INFO: tinylog.level=DEBUG
    2015-09-29 23:20:44 [Test worker] org.tinylog.bugreport.SystemPropertyTest.loggers()
    DEBUG: Log level debug

org.tinylog.bugreport.SystemPropertyTest > loggers PASSED

BUILD SUCCESSFUL

Total time: 1.636 secs
```

Thank you again for your astonishingly quick response. My co-workers were quite impressed with your rapid response.

**Comment by pmwmedia** [comment]
Is fixed in tinylog 1.0.2
